app/models/User.scala

```diff
@@ -0,0 +1,14 @@
+package models
+
+import com.mohiva.play.silhouette.api.{ Identity, LoginInfo }
+
+case class User(
+  userID: Int,
+  firstName: Option[String],
+  credentialId: String,
+  lastName: Option[String],
+  userType: String,
+  email: Option[String]
```

Comment:
`first`, `last`, and `email` are currently not nullable in database. Are model fields supposed to match?